src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -5308,6 +5308,10 @@ and TcExprThen (cenv: cenv) overallTy env tpenv isArg synExpr delayed =
     // f(x)  // hpa=true
     // f[x]  // hpa=true
     | SynExpr.App (hpa, isInfix, func, arg, mFuncAndArg) ->
+        match func with
+        | SynExpr.DotLambda _ -> errorR(Error(FSComp.SR.tcDotLambdaAtNotSupportedExpression(), func.Range))
```

Comment:
I am a bit confused, what case is this type checking? Something like `(_.ToString()) 1`?

Reply:
This for example:

let _ = _.PropertyWhichIsAFunction 123

Reply:
It is an application a function (left) and argument (right), separated with a space.